bson/unmarshal_value_test.go

```diff
@@ -36,60 +36,6 @@ func TestUnmarshalValue(t *testing.T) {
 			})
 		}
 	})
-	t.Run("UnmarshalValueWithRegistry with DefaultRegistry", func(t *testing.T) {
-		t.Parallel()
-
-		for _, tc := range unmarshalValueTestCases {
-			tc := tc
-
-			t.Run(tc.name, func(t *testing.T) {
-				t.Parallel()
-
-				gotValue := reflect.New(reflect.TypeOf(tc.val))
-				err := UnmarshalValueWithRegistry(DefaultRegistry, tc.bsontype, tc.bytes, gotValue.Interface())
-				assert.Nil(t, err, "UnmarshalValueWithRegistry error: %v", err)
-				assert.Equal(t, tc.val, gotValue.Elem().Interface(), "value mismatch; expected %s, got %s", tc.val, gotValue.Elem())
-			})
-		}
-	})
-	// tests covering GODRIVER-2779
-	t.Run("UnmarshalValueWithRegistry with custom registry", func(t *testing.T) {
```

Comment:
Custom registry's can still be defined using `NewDecoder`. Do we currently apply the test matrix for this test to `NewDecoder`. If not, instead of removing this test suggest updating it to the new pattern.

Reply:
Moved to `TestUnmarshalWithRegistry` in "unmarshal_test.go"